src/buttons/FAB.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { StyleSheet, Animated, StyleProp, ViewStyle } from 'react-native';
+import Button, { ButtonProps } from './Button';
+import { IconProps } from '../icons/Icon';
+import { FullTheme, withTheme } from '../config';
+
+export type FABProps = ButtonProps & {
+  iconName?: string;
+  iconColor?: string;
+  iconProps?: IconProps;
+  color?: string;
+  label?: string;
```

Comment:
This gets passed into `title` so just use `title` as given in the `ButtonProps`.